### PR TITLE
Note source-availability

### PR DESCRIPTION
**Pull request by akirataguchi115**
Change open source to source-availability in the README.

Do let me know if I understood business source license wrong here! :octocat:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Alpakka [![scaladex-badge][]][scaladex] [![maven-central-badge][]][maven-central
...
+The Alpakka project is an source-available initiative to implement stream-aware, reactive, integration pipelines for Java and Scala. It is built on top of [Akka Streams](https://doc.akka.io/docs/akka/current/stream/index.html), and has been designed from the ground up to understand streaming natively and provide a DSL for reactive and stream-oriented programming, with built-in support for backpressure. Akka Streams is a [Reactive Streams](http://www.reactive-streams.org/) and JDK 9+ [java.util.concurrent.Flow](https://docs.oracle.com/javase/10/docs/api/java/util/concurrent/Flow.html)-compliant implementation and therefore [fully interoperable](https://doc.akka.io/docs/akka/current/general/stream/stream-design.html#interoperation-with-other-reactive-streams-implementations) with other implementations.
```
Comment (patriknw) [inline README.md:11]: source-available is correct, but looking at other Akka repos I think we can write this without mentioning that. 

Something like
```
The Alpakka project implements stream-aware, reactive...
```

Then we should have a License section at the end of the readme, same as https://github.com/akka/akka#license

**Review by patriknw — COMMENTED** [review]
Thanks, suggestion...

File: README.md (hunk 1 above)
Comment (akirataguchi115) [reply README.md:11]: Good idea! I'll do make that happen.

File: README.md [hunk 2]
```diff
@@ -52,11 +52,13 @@ Refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more details about the
...
 Caveat Emptor
 -------------
 
 Alpakka components are not always binary compatible between releases. API changes that are not backward compatible might be introduced as we refine and simplify based on your feedback. A module may be dropped in any release without prior deprecation. If not stated otherwise, the [Lightbend subscription](https://www.lightbend.com/subscription) does *not* cover support for Alpakka modules.
 
 Our goal is to improve the stability and test coverage for Alpakka APIs over time.
+
+License
+-------
+Akka is licensed under the Business Source License 1.1, see [LICENSE](https://github.com/akka/alpakka/blob/main/LICENSE).
```
Comment (patriknw) [inline README.md:64]:
```suggestion
Alpakka is licensed under the Business Source License 1.1, see [LICENSE](https://github.com/akka/alpakka/blob/main/LICENSE).
```

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**